src/packages/get-ndk.js

```diff
@@ -1,25 +1,13 @@
 const fs = require('fs');
 const path = require('path');
 
-function getNdk() {
-  if (process.env.ANDROID_NDK) {
-    return getNdkVersionFromPath(process.env.ANDROID_NDK)
-  }
-
-  if (process.env.ANDROID_HOME) {
-    return getNdkVersionFromPath(path.join(process.env.ANDROID_HOME, 'ndk-bundle'));
-  }
-
-  return undefined
-}
-
 function getNdkVersionFromPath(ndkDir) {
-  const metaPath = path.join(ndkDir, 'source.properties')
+  const metaPath = path.join(ndkDir, 'source.properties');
   if (fs.existsSync(metaPath)) {
     const contents = fs.readFileSync(metaPath).toString();
-    const split = contents.split("\n");
-    for (const i of split) {
-      const splits = i.split('=');
+    const split = contents.split('\n');
```

Comment:
I think we could probably simplify this quite a bit using a regex with a capture group. 

something like this: `/Pkg.Revision = (\d+\.[\d+|.]+)/g`

Reply:
Maybe later, the file is literally two lines.